spec/index.bs

```diff
@@ -3031,6 +3031,40 @@ interface GPUCommandEncoder {
 GPUCommandEncoder includes GPUObjectBase;
 </script>
 
+{{GPUCommandEncoder}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUCommandEncoder">
+    : <dfn>\[[state]]</dfn> of type {{encoder state}}.
+    ::
+        The current state of the {{GPUCommandEncoder}}, initially set to {{encoder state/open}}.
+</dl>
+
+Each {{GPUCommandEncoder}} has a current <dfn dfn-type="enum">encoder state</dfn> on the [=Content timeline=]
+which may be one of the following:
+
+ - "<dfn dfn-type="enum-value" for="encoder state">open</dfn>" where the {{GPUCommandEncoder}} is available to begin new operations.
+ - "<dfn dfn-type="enum-value" for="encoder state">encoding a render pass</dfn>" where the {{GPUCommandEncoder}} has an active
+    {{GPURenderPassEncoder}}.
+ - "<dfn dfn-type="enum-value" for="encoder state">encoding a compute pass</dfn>" where the {{GPUCommandEncoder}} has an active
+    {{GPUComputePassEncoder}}.
+ - "<dfn dfn-type="enum-value" for="encoder state">closed</dfn>" where the {{GPUCommandEncoder}} is
+     no longer available for any operations.
+
+<div class=note>
```

Comment:
nit: I think I would make this section normative. Doesn't strictly have to be normative as these properties will be encoded elsewhere, but they also would be valid to read as normative.

Could be integrated into the list above if so.

Reply:
Done.